### PR TITLE
Mutliple fixes and introduce automated testing

### PR DESCRIPTION
**Pull request by Gaura**
Major changes:
- Introduce continuous testing with Github actions
- Fix all R CMD check errors and warnings
- Change vignette to HTML from pdf

Other changes:
- Clean up examples
- Remove deprecated functions
- Update versions and imports

Notes:
- The README would need update after merge to point to the checks in the original repo
- Vignette needs fixes with conversion of latex code to show up in html

### CODE REVIEW

**Comment by Gaura** [comment]
Also, data has been recompressed using the most efficient compression for each dataset to avoid warnings.

**Comment by Gaura** [comment]
Hey @genesofeve! Thanks for the merge. Good thing is now the repo has automatic builds. I think there's only [one minor warning remaining](https://github.com/genesofeve/projectR/actions/runs/2630450110). I'll fix it. There are some other changes that need to go in the vignette which I'll add and hopefully get it to you by next week.

**Comment by genesofeve** [comment]
Great! I notice the builds failed as a result of an undefined path variable. That may be worth checking out while your at it. Thank you so much again!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Gaura** [comment]
Hey @genesofeve! So, the error was due to a warning `Warning: requires (indirectly) orphaned package ‘gtools’`, this was nothing do with projectR, but since `ROCR` imports it [checked using `available.packages()`], we got the warning. Now that the `gtools` is not orphaned anymore, a rerun resulted in no error. 

I have updated the badge in README to point to this repo. Also, there are changes in vignette that I need to do because the pdf latex isn't rendering, but those won't result in any error. 

In summary, `projectR` is now error free, few vignette changes pending.
